picasso-provider/src/main/java/com/squareup/picasso3/provider/PicassoProvider.java

```diff
@@ -22,7 +22,7 @@
  */
 public final class PicassoProvider {
   @SuppressLint("StaticFieldLeak")
-  private static Picasso instance;
+  private static volatile Picasso instance;
```

Comment:
hmm. would we ever want this to be accessed off the main thread?

Reply:
Probably not. Just matches what existed before.

Reply:
to match #882